network/vpc/concepts.mdx

```diff
@@ -68,7 +68,7 @@ Private Networks benefit from managed DNS, which resolves the hostnames of attac
 
 ## IPAM
 
-**IP** **A**ddress **M**anagement (IPAM) is a method for planning, tracking and managing the IP address space of a network. Scaleway's internal IPAM acts as a single source of truth for the IP addresses of Scaleway resources. Managed products, such as Databases and Load Balancer, are fully integrated into IPAM, and Private Networks' [DHCP server](#dhcp) uses IPAM to assign static IP addresses to attached resources such as Instances and Elastic Metal server. These IP addresses are fixed, and (for now) cannot be defined by the user. Watch this space for future developments in terms of Scaleway's IPAM and the public functionality, such as IP reservation (be aware that at the time of writing there is no ETA for this).
+**IP** **A**ddress **M**anagement (IPAM) is a method for planning, tracking and managing the IP address space of a network. Scaleway's IPAM acts as a single source of truth for the IP addresses of Scaleway resources. Managed products, such as Databases and Load Balancer, are fully integrated into IPAM, and Private Networks' [DHCP server](#dhcp) uses IPAM to assign static IP addresses to attached resources such as Instances and Elastic Metal server. We are starting to make some public IPAM functionality available on our IPAM API: [check out the documentation](IP reservations are also available through [the IPAM API](https://www.scaleway.com/en/developers/api/ipam/) for more details.
```

Comment:
missing link on `[check out the documentation]` ? and missing parenthesis before "for more details" ?